main/src/main/java/cgeo/geocaching/apps/navi/AbstractPointNavigationApp.java

```diff
@@ -36,24 +36,30 @@ protected AbstractPointNavigationApp(@NonNull final String name, @Nullable final
     }
 
     @Override
-    public void navigateWithoutSelector(@NonNull final Context context, @NonNull final Geocache cache) {
```

Comment:
By reversing the logic from `navigateWithoutSelector` back to `navigate` navigation you'll end up in a loop if you select the geocache (not one of its waypoints) as navigation target.